app/models/lookup_key.rb

```diff
@@ -158,6 +158,10 @@ def path_elements
     end
   end
 
+  def contains_erb?(value)
```

Comment:
maybe not this but we should add the check to validating after ENC evaluation you mention below, I'll look at this